code/modules/clothing/suits/medium_armor.dm

```diff
@@ -320,6 +320,16 @@
 // BROTHERHOOD OF STEEL //
 //////////////////////////
 
+/obj/item/clothing/suit/armored/medium/combat/flatline/prototype/bos
+	name = "Brotherhood 'Flatline' Armor"
+	desc = "A modified and refined self-revival 'flatline' armor developed by none other than flatline himself for his pack member he cares so much for. Repainted to the color scheme of the Brotherhood of Steel. Its lining inside is a gel-based refinement of the symbiotic algae found in buffalo gourds giving it flesh-mending properties over time. Inside it is a regulated and biomonitor-controlled self-defibrilator unit to bring the host back to life after the gel has done its job."
```

Comment:
Is "flatline" a name? If so, capitalize it.
```suggestion
	desc = "A modified and refined self-revival 'flatline' armor developed by none other than Flatline himself for his pack member he cares so much for. Repainted to the color scheme of the Brotherhood of Steel. Its lining inside is a gel-based refinement of the symbiotic algae found in buffalo gourds giving it flesh-mending properties over time. Inside it is a regulated and biomonitor-controlled self-defibrillator unit to bring the host back to life after the gel has done its job."
```

Reply:
Also, highly dislike having character backstory in item descriptions. How would someone know all this just from examining the armor? Especially if they've stolen it, or taken it off of a corpse?